### ISSUE TITLE
[aws-cdk] API Reference doc not available for older version 

### ISSUE THREAD

**Issue by flochaz** [aws/aws-cdk#11168]
## TL;DR;

https://docs.aws.amazon.com/cdk/api/latest/ website only expose the `latest` version. 

## Details

CDK is evolving at a fast pace with minor version update regularly. As such the documentation is also evolving and the latest version is the only available as a hosted website. We noticed that some construct need to stick to previous versions for backward compatibility issue but the doc for the previous versions are not available on [aws web site](https://docs.aws.amazon.com/cdk/api/latest/) and need to be manually pull from old tag from the aws-cdk github report and rebuilt locally. Customers could not find how CDK was working previously, conducting to poor user experience.

---

This is a 📕 documentation issue

**Comment by SomayaB** [comment]
Duplicate of https://github.com/aws/aws-cdk/issues/11172
